Gtk-4.0.gir

```diff
@@ -84187,6 +84205,9 @@ The order in which the widgets are returned is not defined.</doc>
       <field name="segment" readable="0" private="1">
         <type name="gpointer" c:type="gpointer"/>
       </field>
+      <field name="chars" readable="0" private="1">
+        <type name="utf8" c:type="char*"/>
+      </field>
```

Comment:
This would be an ABI change... but there is no `chars` field here: https://gitlab.gnome.org/GNOME/gtk/-/blob/master/gtk/gtktextchild.h#L62

What happened? :)

Reply:
it's here https://gitlab.gnome.org/GNOME/gtk/-/blob/main/gtk/gtktextchild.h#L63? you're looking at the master branch which is no longer supposed to be used.

Reply:
Oh I see. Thanks. Reported https://gitlab.gnome.org/GNOME/gtk/-/issues/4531